framework/src/modules/chain/blocks/block.js

```diff
@@ -31,6 +31,14 @@ const storageReadFunc = {
 };
 const storageRead = raw => storageReadFunc[raw.version](raw);
 
+const objectNormalizeFunc = {
+	0: blockV1.objectNormalize,
```

Comment:
I thought this logic would be moved to the processor and not stay inside blocks (block_v1, block_v2)?

Reply:
It has another issue #4131 to address this